examples/storybook/src/stories/RangeSlider.stories.ts

```diff
@@ -0,0 +1,14 @@
+import { storiesOf } from '@storybook/angular';
+import { wrapWithHits } from '../wrap-with-hits';
+import meta from '../meta';
+
+storiesOf('RangeSlider', module)
+  .addDecorator(meta)
+  .add('default', () => ({
+    component: wrapWithHits({
+      template: `
+      <ais-range-slider attribute="price">
```

Comment:
The `RangeSlider` doesn't work with preview.

Reply:
fixed here:  ac80fb3